vmm/src/memory_manager.rs

```diff
@@ -86,6 +100,7 @@ pub struct MemoryManager {
     sgx_epc_region: Option<SgxEpcRegion>,
     use_zones: bool,
     snapshot_memory_regions: Vec<MemoryRegion>,
+    numa_nodes: NumaNodes,
```

Comment:
Thanks for you patch, i would like to understand why would not place it to `struct vm`,  since numa_node seems like it's not the member of MemoryManager, it also has CPU topology.

Reply:
I had a discussion with @sameo  and he thought `MemoryManager` was more appropriate. I don't have a strong opinion on this, and if needed, we can move it to the `Vm` structure in the future if we realize it was not a good design.

Reply:
Thanks for you kind reply. :+1: